torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1575,8 +1680,10 @@ void TensorExprKernel::compile() {
     if (has_symbolic_shapes_) {
       auto sizes = sizesFromSymbolicShape(tt->symbolic_sizes());
       tensorOutputSymbolicSizes_.push_back(sizes);
-      TORCH_INTERNAL_ASSERT(sym_stride_outputs_.count(i));
-      auto stride_desc = sym_stride_outputs_[i];
+      TORCH_INTERNAL_ASSERT(symbolic_strides_.count(output));
+      auto stride_desc_vec = symbolic_strides_[output];
+      TORCH_INTERNAL_ASSERT(stride_desc_vec.size() == 1);
```

Comment:
If the size of the desc_vec is always 1, why is it even a vector? In what scenarios can it have more than 1 element?

Reply:
If the tensor is contiguous or channel-last contiguous, its symbolic strides always only contains 1 element - `[TENSOR_CONT]` or `[TENSOR_CONT_CHANNELS_LAST]`. But if the tensor is non-contiguous, its symbolic strides have more than 1 element. Take an example as follows, the symbolic stride is `[S_CONT, S_CONT, S_AS_ARG, S_AS_ARG, S_ONE, S_TRAN_CONT]`
```python
x = torch.randn(3, 2, 5, 4, 6, 7)
x = x.permute(1, 2, 3, 0, 5, 4)
```